### ISSUE TITLE
Compare/equal

### ISSUE THREAD

**Issue by alanshaw** [rvagg/bl#68]
Any plans/wants for comparing buffer lists without having to convert to buffer and then compare?

**Comment by mcollina** [comment]
It would be awesome to get a PR!

**Comment by rvagg** [comment]
Agreed. I'd love to keep a decent level of compatibility with the Buffer API but it requires time and effort!
Does this happen to be for IPFS @alanshaw? Perhaps it's easier for me to justify putting in some effort if it is.

**Comment by alanshaw** [comment]
> Does this happen to be for IPFS @alanshaw? Perhaps it's easier for me to justify putting in some effort if it is.

I can't categorically say yes yet. I would definitely use it in tests right now, and it will very likely be useful in the future for production code. It just struck me as something missing that others might want also. I'm experimenting with using `bl` throughout the libp2p stack as it is converted to using async/await/async iterators.

**Comment by rvagg** [comment]
Closing as stale
